smg-automotive/components-pkg
# fix: remove transparent button variant bg color

References [JIRA ticket](https://autoricardo.atlassian.net/browse/FE-184)

## Motivation and context

When using a transparent button variant in a component that has its own background colour, the button shows white background and on hover its background differs from the parent component.

## Before

Transparent button variant shows white background as a default state.

## After

Transparent button variant shows transparent background as a default state.

## How to test

Use Storybook to access button variants and confirm it works as expected.

https://autoricardo.atlassian.net/browse/FE-184
